lib/param_parsing/parameters.go

```diff
@@ -52,32 +52,33 @@ func GetParameters() Params {
 	// Parse the command line parameters to fetch stuff like chdir
 	getopt.Parse()
 
-	oldLogLevel := params.LogLevel
-	logger = lib.InitLogger(params.LogLevel)
-	var err error
-	// Read configuration files
-	if tomlFileExists(params) {
-		params, err = getParamsTOML(params)
-	} else if tfSwitchFileExists(params) {
-		params, err = GetParamsFromTfSwitch(params)
-	} else if terraformVersionFileExists(params) {
-		params, err = GetParamsFromTerraformVersion(params)
-	} else if isTerraformModule(params) {
-		params, err = GetVersionFromVersionsTF(params)
-	} else if terraGruntFileExists(params) {
-		params, err = GetVersionFromTerragrunt(params)
-	} else {
-		params = GetParamsFromEnvironment(params)
-	}
-	if err != nil {
-		logger.Fatalf("Error parsing configuration file: %q", err)
-	}
-
-	// Logger config was changed by the config files. Reinitialise.
-	if params.LogLevel != oldLogLevel {
+	if !params.VersionFlag {
```

Comment:
Should we add all of the params I listed at https://github.com/warrensbox/terraform-switcher/issues/443#issuecomment-2147257097 ? 🤔

Reply:
That would be complicated. We would need to parse the options a third time.
I don't see a problem in parsing the other variables though.

Reply:
> That would be complicated.

Ah, that's weird 😢 
* `params.HelpFlag`
* `params.LatestFlag`
* `params.LatestPre`
* `params.LatestStable`
* `params.ListAllFlag`
* `params.ShowLatestFlag`
* `params.ShowLatestPre`
* `params.ShowLatestStable`
* `params.VersionFlag`

It's just pointless to find TF version for these options.

Since this would require more effort, let's go with baby steps then =)